recipes/freetype/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.5.2" %}
+
+package:
+    name: freetype
+    version: {{ version }}
+
+source:
+    fn: freetype-{{ version }}.tar.bz2
+    url: http://download.savannah.gnu.org/releases/freetype/freetype-{{ version }}.tar.bz2
+    sha256: 4ff4bd393aa01071ec7b849d035508a505a78f88b2bcf25ff11e58e43c0b9def
+    patches:
+        - subpixel.patch
+        - multi-thread-violations.patch
+        - double-free.patch
+        - CVE-2014-2240.patch
+        - CVE-2014-2241.patch
```

Comment:
Sorry, it's a possible problem with patents:

https://www.freetype.org/patents.html

In particular, take a look at the last section. It doesn't seem so serious though :-)